test/parallel/test-http-extra-response.js

```diff
@@ -51,7 +51,7 @@ const server = net.createServer(function(socket) {
 
     if (postBody.includes('\r\n')) {
       socket.write(fullResponse);
-      // omg, I wrote the response twice, what a terrible HTTP server I am.
+      // I wrote the response twice, what a terrible HTTP server I am.
```

Comment:
I don't know if this comment adds much in general, to be honest. I would be in favour of taking it out.